### PR TITLE
Food Website

### PR DESCRIPTION
**Pull request by apu52**
### **Describe the project you want to add with tech stack**
This repo contains my project on my Food Website😋. I have made it using HTML,CSS & **JS (mainly)**. It contains a menu card of different cuisines such as Biriyani , Chinese, Paneer Mania, Veg as well as South Indian.🍕 . 

### **Expected behavior**
Here is my Food website of the resturent names Mr. Chef(ESTD. 2003)🍴 .This website is responsive & it has been made using HTML, CSS, JS. Here are some key features of the website.

- A menu card of different cuisines such as Biriyani , Chienese, Paneer Mania, Veg as well as South Indian.🍕
- A scroll or jump to section to different sections of menu card🥗
- Every dish has it's own rating & prices🥘
- A very unique Add to cart system ( customers can also see the total billing there) 🍟
- Address Saving Option🦑🦐

### **Screenshots**

image

### Additional context (optional )

Please take a review of my PR & merge it   under HACTOBERFEST 2023 .

### CODE REVIEW

**Comment by apu52** [comment]
@sadanandpai  please take a look in my PR & after reviewing , Please merge it under HACTOBERFERST 2023.

**Comment by apu52** [comment]
@sadanandpai please take a look in my PR & after reviewing , Please merge it under HACTOBERFERST 2023.

File: native/machine-coding/Food Website/index.js
Comment (sadanandpai) [inline native/machine-coding/Food Website/index.js:1]: Code can be refactored to to keep it maintainable and simple.

- Use array and loop over if needed
- create reusable functions and make calls
- prefer let over var
- add comments wherever necessary

**Review by sadanandpai — CHANGES_REQUESTED** [review]

File: native/machine-coding/Food Website/index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0,viewport-fit=cover">
+    <title>Food Website</title>
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
```
Comment (sadanandpai) [inline native/machine-coding/Food Website/index.html:10]: import the header as done in [counter](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/native/machine-coding/counter/index.html)

Also, remove the other tags

File: native/machine-coding/Food Website/index.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0,viewport-fit=cover">
+    <title>Food Website</title>
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (sadanandpai) [inline native/machine-coding/Food Website/index.html:9]: dow we really need font awesome

File: native/machine-coding/Food Website/index.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        </div>
+
+        <div class="mobile-footer">
+            <p>Home</p>
+            <p>Cart</p>
+            <p>offers</p>
+            <p>orders</p>
+        </div>
+    </div>
+    <script src="/index.js" type="module"></script>
```
Comment (sadanandpai) [inline native/machine-coding/Food Website/index.html:156]: move inside head

File: native/machine-coding/Food Website/index.js [hunk 4]
```diff
@@ -0,0 +1,497 @@
+import {foodItem} from './fooditem.js'
+
+function displayItems(){
+    var biryani= document.getElementById('biryani');
+    var paneer=  document.getElementById('paneer');;
+    var chicken=  document.getElementById('chicken');
+    var vegetable=  document.getElementById('vegetable');
+    var chinese=  document.getElementById('chinese');
+    var southIndian=  document.getElementById('south-indian');
```
Comment (sadanandpai) [inline native/machine-coding/Food Website/index.js:9]: use const or let as per the usecase

File: native/machine-coding/Food Website/style.css [hunk 5]
```diff
@@ -0,0 +1,524 @@
...
+        padding: 8px 15px;
+        margin: auto;
+    }
+    .btn-box{
+        text-align: center;
+    }
+    
+}
+
+@media screen and (max-width : 400px) and (-webkit-min-device-pixel-ratio: 2){
```
Comment (sadanandpai) [inline native/machine-coding/Food Website/style.css:513]: any reason for -webkit-min-device-pixel-ratio: 2

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Pull request closed without merging**